src/main/java/ua/javaPro/berezhnoy/hillelHomeWork7/FileNavigator.java

```diff
@@ -0,0 +1,55 @@
+package ua.javaPro.berezhnoy.hillelHomeWork7;
+
+import java.util.*;
+import java.util.List;
+
+public class FileNavigator {
+    private Map<String, List<FileData>> files = new HashMap<>();
+
+    public void add(String path, FileData newFile) throws MyException {
+
+        if (!path.equals(newFile.getPath())) {
+            throw new MyException("Wrong way");
```

Comment:
```suggestion
            throw new MyException("Wrong path");
```
Англійський path краще підходить в нашому контексті.